resources/cgi-bin/scrollery-cgi.pl

```diff
@@ -19,6 +19,52 @@ sub processCGI {
 		exit;
 	}
 
+	my %actions = (
```

Comment:
Bummer that we need to bring this back. Is it something about Perl versions?

Reply:
I think it is because the perl script is run by mod_perl on the server, which changes the context of "main."  Thus $:: no longer points to scrollery-cgi.pl and it cannot find the referenced subroutines.